src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,15 +2,21 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
+        logger.info("Login method was called with login: {}", login);
         //TODO: add corresponding log message about method login was called
```

Comment:
The `TODO` comment suggests adding a log message about the login method being called. This is already partially implemented with the existing log statement. Ensure that the log message is complete and provides all necessary information as per the task requirements.